test/integration/mapping_validation_test.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require_relative "../test_helper"
+require_relative "../../config/cli"
+
+class LocalizationsTest < ActiveSupport::TestCase
+  test "category IDs in mappings are valid" do
+    mappings_json_data = CLI.new.parse_json("dist/en/integrations/all_mappings.json")
+    invalid_categories = []
+    mappings_json_data["mappings"].each do |mapping|
+      # process input taxonomy
+      category_ids = get_category_ids(mapping["input_taxonomy"])
```

Comment:
nit: I'd move this into `validate_mapping_category_ids` since you already have to pass in `mapping["input_taxonomy"]` there.

Reply:
Done